### ISSUE TITLE
Release v1.50.7

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#5764]
:robot: A dedicated issue to run tests and deploy release [v1.50.7](https://github.com/deckhouse/deckhouse/milestone/227).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.50.7 | test-v1.50.7*` or a branch `main | release-1.50`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `docker | containerd`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.23 | 1.24 | 1.25 | 1.26 | 1.27`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.50.7 | test-v1.50.7*` or a branch `main | release-1.50`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.25'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.20
/e2e/use/k8s/1.23

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/cri/containerd
/e2e/use/k8s/1.23

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.50
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [f6bd4c](https://github.com/deckhouse/deckhouse/commit/f6bd4cce213b65cf5665230ff979bbc1355af9eb) in branch **main**:
```
Fix connect madison registration (#5765)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b98ad0](https://github.com/deckhouse/deckhouse/commit/b98ad0b07775b4736d1228c3caef7c45309da39d) in branch **main**:
```
[common] common images build from own repo (#5750)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [9902cf](https://github.com/deckhouse/deckhouse/commit/9902cf2a8ce999c6dd425337750b0cdf87788890) in branch **main**:
```
[user-authn][docs] Update description of the namespaceSelector parameter (#5781)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4dbcce](https://github.com/deckhouse/deckhouse/commit/4dbccebf01a038e5bfe37c996cb396dc4a5b8827) in branch **main**:
```
[candi] Rename vendored containerd.service into containerd-deckhouse.service (#5677)

Signed-off-by: Maxim Vasilenko <maksim.vasilenko@flant.com>
Co-authored-by: Maxim Vasilenko <maksim.vasilenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [49ec85](https://github.com/deckhouse/deckhouse/commit/49ec8586aea163719a86ec827f2739830b2ce67b) in branch **main**:
```
[node-manager][deckhouse-controller] bashible-apiserver readiness probe tied to deckhouse readiness (#5722)

Signed-off-by: Maxim Vasilenko <maksim.vasilenko@flant.com>
Co-authored-by: Maxim Vasilenko <maksim.vasilenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [774a71](https://github.com/deckhouse/deckhouse/commit/774a718016ce02e12675a793112870af0d0c7621) in branch **main**:
```
Distroless prometheus (#5783)

Signed-off-by: Alexey Sulimenko <alexey.sulimenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [05a4e6](https://github.com/deckhouse/deckhouse/commit/05a4e6c98fa3a40098163a3e8cc66d7e927ee553) in branch **main**:
```
[cloud-provider-openstack] Fix ignoreVolumeMicroversion parameter appliance for k8s>1.24 (#5778)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [46b24f](https://github.com/deckhouse/deckhouse/commit/46b24f53151712e106c2f8154ec9451176969d63) in branch **main**:
```
Re-generate changelog v1.51.0 (#5788)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [fd3eeb](https://github.com/deckhouse/deckhouse/commit/fd3eeb3b712599e5fb8285175884a1e78c456b1b) in branch **main**:
```
[external-module-manager] Fix panic if there are no resources for migration (#5793)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2eef11](https://github.com/deckhouse/deckhouse/commit/2eef1116655571ead14368d115d3382aa9a5c40c) in branch **main**:
```
Re-generate changelog v1.51.0 (#5795)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [fb0e53](https://github.com/deckhouse/deckhouse/commit/fb0e53b0a356c1f3653d597a7107d820eabcd871) in branch **main**:
```
[log-shipper] fix Dockerfile (#5794)

Signed-off-by: Alexey Sulimenko <alexey.sulimenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [7995c3](https://github.com/deckhouse/deckhouse/commit/7995c3af90be5ccdf071e6363eee9460aef66614) in branch **main**:
```
[prometheus] Change description and formula for oldest metrics panel (#5712)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [bd06d8](https://github.com/deckhouse/deckhouse/commit/bd06d85bf5bf1a60bdda2ce21b99cb720e9da500) in branch **main**:
```
Fix openapi validation (#5801)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2ec550](https://github.com/deckhouse/deckhouse/commit/2ec550c677c41f4b21984e02c371eddd912bd0a1) in branch **main**:
```
[runtime-audit-engine] Improve documentation (#5168)

Signed-off-by: Artem Kulyabin <artem.kulyabin@flant.com>
Co-authored-by: Nikolay Mitrofanov <30695496+name212@users.noreply.github.com>
Co-authored-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ae9c97](https://github.com/deckhouse/deckhouse/commit/ae9c97ac080018caf789e9e69a2c7fa2e4ef62b5) in branch **main**:
```
[runtime-audit-engine] Add fstec falco rules (#5787)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
Signed-off-by: Denys Romanenko <65756796+RomanenkoDenys@users.noreply.github.com>
Co-authored-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [e05626](https://github.com/deckhouse/deckhouse/commit/e05626d136daad3d0f23aa80fd75e62c7c83a95e) in branch **main**:
```
[delivery] Fix customCertificate (#5762)

Signed-off-by: Andrey Sidorov <andrey.sidorov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4049ba](https://github.com/deckhouse/deckhouse/commit/4049ba9d8d881d07497dfee93192a7624bf6d270) in branch **main**:
```
[docs] Update content of the getting started

Signed-off-by: VV-corrector <144900662+VV-corrector@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b31df6](https://github.com/deckhouse/deckhouse/commit/b31df6e94dfd1f8385b979de8207d9c3962a8801) in branch **main**:
```
[guide] Update LB diagram (#5819)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ea38c0](https://github.com/deckhouse/deckhouse/commit/ea38c07b350646add7b9cd3510c246de2a1b43b5) in branch **main**:
```
[deckhouse] Fix deckhouse port (#5815)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [7039d0](https://github.com/deckhouse/deckhouse/commit/7039d0d9bb41a72cd3ab9749f75c561b496bbe5e) in branch **main**:
```
[local-path-provisioner] Rewrite build to werf and use deckhouse user (#5809)

Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [da0da8](https://github.com/deckhouse/deckhouse/commit/da0da816733014385cfb340e70c4ad409e395f2f) in branch **main**:
```
[user-authn] Distroless (#5512)

Signed-off-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [29c070](https://github.com/deckhouse/deckhouse/commit/29c070e3ea326e78abd1d5377c7b59130db231b3) in branch **main**:
```
[log-shipper] Supress metrics timestamp (#5835)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [e4f498](https://github.com/deckhouse/deckhouse/commit/e4f4986f384eb9327cf37b42390513b32704926b) in branch **main**:
```
[control-plane-manager] Distroless (#5509)

Signed-off-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [744fa7](https://github.com/deckhouse/deckhouse/commit/744fa716e5d8ab2b73044b847f05425cc3ad9c6b) in branch **main**:
```
[registry-packages] refactor build (#5798)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [509a5d](https://github.com/deckhouse/deckhouse/commit/509a5d8376c856a639f9c00d0b45e66c5504ed55) in branch **main**:
```
[prometheus] fix issue with add custom grafana plugins (#5834)

Signed-off-by: Vitaliy Snurnitsin <vitaliy.snurnitsin@flant.com>
Signed-off-by: Alexey Sulimenko <alexey.sulimenko@flant.com>
Co-authored-by: Alexey Sulimenko <alexey.sulimenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [f5f40e](https://github.com/deckhouse/deckhouse/commit/f5f40e16656fb2e2f0807706d3a586b4136eb196) in branch **main**:
```
[docs] Proofreading (#5848)

Signed-off-by: VV-corrector <144900662+VV-corrector@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [bb03f6](https://github.com/deckhouse/deckhouse/commit/bb03f65e8ec6d67756914cc25a051b2606ca6825) in branch **main**:
```
[docs] Proofreading (#5852)

Signed-off-by: VV-corrector <144900662+VV-corrector@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3b356d](https://github.com/deckhouse/deckhouse/commit/3b356d2eb5dcee3585bf191a5a633caa7dcda77d) in branch **main**:
```
[monitoring] Fixing the etcd version in the home dashboard (#5842)

Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b69ba5](https://github.com/deckhouse/deckhouse/commit/b69ba53d28e733bed2870e558b4acdc5cbaf3960) in branch **main**:
```
Re-generate changelog v1.51.1 (#5760)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [61360e](https://github.com/deckhouse/deckhouse/commit/61360ed628919234fabf6cd07d0a13bbef3d1a90) in branch **main**:
```
[testing] Fix template for vSphere (#5862)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```
